app/Auth/TotpAuth.php

```diff
@@ -117,13 +117,13 @@ public function getSecret()
      * @param  string $label
      * @return string
      */
-    public function getQrCodeUrl($label)
+    public function getQrCodeUrl($label, $options)
```

Comment:
If you add a new argument, you should update the phpdoc of the method and specify the type if not scalar (array).